save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPluginConfig.kt

```diff
@@ -60,10 +60,18 @@ data class WarnPluginConfig(
             this.lineCaptureGroup ?: other.lineCaptureGroup,
             this.columnCaptureGroup ?: other.columnCaptureGroup,
             this.messageCaptureGroup,
-            this.exactWarningsMatch ?: other.exactWarningsMatch
+            this.exactWarningsMatch ?: other.exactWarningsMatch,
+            this.testNameSuffix ?: other.testNameSuffix
         )
     }
 
+    private fun resourceNamePattern(): Regex = testNameSuffix?.let {
+        Regex("""(.+)$testNameSuffix\.[\w\d]+""")
+    }
+        ?: run {
+            Regex("""(.+)Test\.[\w\d]+""")
+        }
```

Comment:
```suggestion
    private fun resourceNamePattern(): Regex = (testNameSuffix ?: "Test").let {
        Regex("""(.+)$it\.[\w\d]+""")
    }
```
I'd write like this, in order not to repeat regex twice